config/m4/gdrcopy.m4

```diff
@@ -49,7 +49,7 @@ AS_IF([test "x$with_gdrcopy" != "xno"],
             ],
             [
                 AS_IF([test "x$with_gdrcopy" != "xguess"],
-                    [AC_MSG_ERROR([gdrcopy support is requested but gdrcopy packages can't found])],
+                    [AC_MSG_ERROR([gdrcopy support is requested but gdrcopy packages can't be found])],
```

Comment:
minor: [AC_MSG_ERROR gnu page](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Printing-Messages.html) says that "cannot" is prefered to "can't"

Reply:
Yes, indeed.  Focused on the "be"